components/org.wso2.carbon.editor.log.appender/pom.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ ~ Copyright (c) WSO2 Inc. (http://wso2.com) All Rights Reserved.
```

Comment:
Shall we add the new licence header with the year?

Reply:
@NisalaNiroshana : This was merged on 2017. I hope this isn't needed unless the file gets changed again.

Reply:
Fixed in commit [#99bc41d8e251e1a1528823b6781f09194a4cc731](https://github.com/wso2/carbon-analytics/pull/1259/commits/99bc41d8e251e1a1528823b6781f09194a4cc731)